ingestor/chalicelib/service_ridership_dashboard/util.py

```diff
@@ -0,0 +1,69 @@
+from typing import Tuple
+from datetime import datetime, date, timedelta
+
+
+def date_from_string(date_str):
+    return datetime.strptime(date_str, "%Y-%m-%d").date()
+
+
+def date_to_string(date):
+    return date.strftime("%Y-%m-%d")
+
+
+def index_by(items, key_getter):
+    res = {}
+    if isinstance(key_getter, str):
+        key_getter_as_str = key_getter
+        key_getter = lambda dict: dict[key_getter_as_str]  # noqa: E731
```

Comment:
Instead of disabling each line, we can disable this rule repo wise if it's not helpful